hangman/dictionary/.gitignore

```diff
@@ -0,0 +1,20 @@
+# The directory Mix will write compiled artifacts to.
```

Comment:
## -5

You have copied the whole dictionary project underneath the hangman project. That's not a good idea. Previously we've looked at doing this by just using 

~~~ elixir
def deps do
  [
    { dictionary: path: "../dictionary" }
  ]
end
~~~

That way is is shared and not restricted to hangman.